diff --git a/src/nsterm.m b/src/nsterm.m
index 507f2a9e7d..5fa8efef2c 100644
--- a/src/nsterm.m
+++ b/src/nsterm.m
@@ -9435,7 +9435,7 @@ - (id)accessibilityAttributeValue:(NSString *)attribute
   NSTRACE ("[EmacsWindow accessibilityAttributeValue:]");
 
   if ([attribute isEqualToString:NSAccessibilityRoleAttribute])
-    return NSAccessibilityTextFieldRole;
+    return NSAccessibilityWindowRole;
 
   if ([attribute isEqualToString:NSAccessibilitySelectedTextAttribute]
       && curbuf && ! NILP (BVAR (curbuf, mark_active)))
